juniper_official/Solutions/Pfe-wedge-detection/lueachip-pattern-set.rule

```diff
@@ -0,0 +1,69 @@
+healthbot {
+    ingest-settings {
+        syslog {
+            pattern LUEA_EVENT001 {
+                event-id LUEA_EVENT001;
+                filter "fpc%{NUMBER:fpc_number}.*lmem addr error";
+                description "The \"lmem addr error\" message is caused by a transient hardware or software defect. This error is caused by a transient hardware or software defect. ";
+                constant syslog-trigger-color {
+                    value red;
+                }
+                constant syslog-trigger-message {
```

Comment:
And here..

Reply:
@ConstrainedRandomVar , added trigger based on count